src/cranbtree.c

```diff
@@ -46,6 +46,26 @@
  */
 static void destroy_bt_helper(cbt_node_t * root, int n, void (*done) (void *));
 
+/*
+ * errors
+ */
+
+static char *errorMessages[] = {
+	NULL,
+	"Cannot perform this operation on a cloned tree",
+};
+
+/*
+ * cranbtree_t* -> const char*
+ * Return pointer to string describing last error. NULL if no error recorded.
+ */
+const char *cbt_error(cranbtree_t * bt)
```

Comment:
same as the above. The error codes will be included in the documentations so that the users can know which code belongs to which error.